ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/patch-panel.j2

```diff
@@ -0,0 +1,29 @@
+{% if patch_panel is arista.avd.defined %}
+
+## Patch Panel
+
+### Patch Panel Summary
+
+| Patch Name | Connector 1 | Connector 2 | Enabled |
```

Comment:
Instead of adding strings into the values for type, I suggest you create a table field for Connector 1 Type etc.

Reply:
Fixed.